.circleci/ct-config.yaml

```diff
@@ -1,3 +1,4 @@
 ---
 chart-repos:
   - cluster-catalog=https://giantswarm.github.io/cluster-catalog
+  - cluster-test-catalog=https://giantswarm.github.io/cluster-test-catalog
```

Comment:
Was this a mistake?

Reply:
I tested using a `wip` version that was pushed to the test catalog, but that made `abs` to fail unless I added that. I don't think it makes any harm?